From 16fe939eeb53d5a5c5478e37e2c8a12b70a709ef Mon Sep 17 00:00:00 2001
Message-Id: <16fe939eeb53d5a5c5478e37e2c8a12b70a709ef.1349175436.git.minovotn@redhat.com>
In-Reply-To: <94968b7fa9b14e71f004474d7ce77e189e6a2bf3.1349175436.git.minovotn@redhat.com>
References: <94968b7fa9b14e71f004474d7ce77e189e6a2bf3.1349175436.git.minovotn@redhat.com>
From: Eduardo Habkost <ehabkost@redhat.com>
Date: Fri, 28 Sep 2012 16:50:06 +0200
Subject: [PATCH 02/34] x86/cpuid: add "host" to the list of supported CPU
 models

RH-Author: Eduardo Habkost <ehabkost@redhat.com>
Message-id: <1348851023-31907-2-git-send-email-ehabkost@redhat.com>
Patchwork-id: 42504
O-Subject: [RHEL6 qemu-kvm PATCH 01/18] x86/cpuid: add "host" to the list of supported CPU models
Bugzilla: 833152
RH-Acked-by: Alex Williamson <alex.williamson@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Igor Mammedov <imammedo@redhat.com>
RH-Acked-by: Orit Wasserman <owasserm@redhat.com>

From: Andre Przywara <andre.przywara@amd.com>

Bugzilla: 833152

Signed-off-by: Andre Przywara <andre.przywara@amd.com>
Signed-off-by: Aurelien Jarno <aurelien@aurel32.net>
(cherry picked from commit ed2c54d4ccfc1f64b174712d0fc02dbe35f75744)

[ehabkost: including this patch only to reduce conflicts when pulling
other "-cpu" changes]

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 target-i386/cpuid.c | 3 +++
 1 file changed, 3 insertions(+)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 target-i386/cpuid.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/target-i386/cpuid.c b/target-i386/cpuid.c
index f780886..8b77d05 100644
--- a/target-i386/cpuid.c
+++ b/target-i386/cpuid.c
@@ -771,6 +771,9 @@ void x86_cpu_list (FILE *f, int (*cpu_fprintf)(FILE *f, const char *fmt, ...),
             (*cpu_fprintf)(f, "\n");
         }
     }
+    if (kvm_enabled()) {
+        (*cpu_fprintf)(f, "x86 %16s\n", "[host]");
+    }
 }
 
 int cpu_x86_register (CPUX86State *env, const char *cpu_model)
-- 
1.7.11.4

